docs/source/release_notes.rst

```diff
@@ -12,6 +12,7 @@ Future Release
         * Add DateToHoliday Transform Primitive (:pr:`1848`)
         * Add DistanceToHoliday Transform Primitive (:pr:`1853`)
         * Temporarily restrict pandas and koalas max versions (:pr:`1863`)
+        * Split date, time, latitude and longitude primitives into separate files (:pr:`1861`)
```

Comment:
This is a very minor nitpick, but I'd probably say `Split Datetime and LatLong primitives into separate files` here to make it a little more clear we are talking about primitives that specifically operate on `Datetime` or `LatLong` column inputs. Not a big deal though.